src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/InsertOperation.java

```diff
@@ -159,28 +170,30 @@ private static Uni<DocumentId> insertDocument(
   }
 
   // utility for building the insert query
-  private String buildInsertQuery(boolean vectorEnabled) {
+  public String buildInsertQuery(boolean vectorEnabled) {
     if (vectorEnabled) {
       String insertWithVector =
           "INSERT INTO \"%s\".\"%s\""
               + " (key, tx_id, doc_json, exist_keys, array_size, array_contains, query_bool_values, query_dbl_values , query_text_values, query_null_values, query_timestamp_values, query_vector_value)"
               + " VALUES"
-              + " (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)  IF NOT EXISTS";
+              + " (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) "
+              + (offlineMode ? "" : " IF NOT EXISTS");
```

Comment:
Should be explained somewhere why "IF NOT EXISTS" not used in offlineMode?
(not needed as we are creating new sstables and not modifying existing tables?)

Reply:
Added comment in this commit https://github.com/stargate/data-api/pull/905/commits/af4044312cc8c08c6351667c7c7b89a657fa00fd